osu.Game.Rulesets.Mania/Mods/ManiaModFadeIn.cs

```diff
@@ -12,5 +13,7 @@ public class ManiaModFadeIn : ManiaModHidden
         public override string Acronym => "FI";
         public override IconUsage? Icon => OsuIcon.ModHidden;
         public override string Description => @"Keys appear out of nowhere!";
+
+        protected override PlayfieldCoveringContainer CreateCover() => new PlayfieldCoveringContainer();
```

Comment:
maybe deserves a comment about why this uses the default while the less derived class doesn't.